modules/costs/app/components/time_entries/time_entry_form_component.html.erb

```diff
@@ -28,20 +28,20 @@ Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
   ++# %>
 
   <%= component_wrapper do
-    primer_form_with(**form_options) do |form|
-      flex_layout(classes: "time-entry-dialog-form-spacing") do |body|
-        body.with_row do
-          render(TimeEntries::UserForm.new(form, visible: show_user))
+        primer_form_with(**form_options) do |form|
+          flex_layout(classes: "time-entry-dialog-form-spacing") do |body|
+            body.with_row do
+              render(TimeEntries::UserForm.new(form, visible: show_user))
+            end
+            body.with_row { render(TimeEntries::DaysAndHoursForm.new(form)) }
+            body.with_row(display: show_work_package ? :block : :none) do
```

Comment:
same problem appear in cases where we do not show the user field. We can do that here as well, let me add it